### DISCUSSION TITLE
Question(s)

### DISCUSSION THREAD

**Discussion by Fireworkstars46** [Dhranios/Luigi-s-Mansion#65] [Q&A] [answered]
Did my post get deleted? Cause I don't see it anymore

**Comment by Fireworkstars46** [comment]
And also can I use optifine with the map or is it not compatible?

**Comment by Dhranios** [comment, marked as answer]
The previous post was an issue entry, which was closed long ago, and discussions should've been moved here.

Optifine should work fine with the map.

**Comment by Fireworkstars46** [comment]
As for optifine, what settings do I set it to so the game works good and also not too laggy?

**Reply by Dhranios** [reply]
It should work with any settings (if it doesn't, make a bug report)

**Comment by Fireworkstars46** [comment]
Also I saw the faqs and it said it's multiplayer compatible, is it true or no? If it is, how exactly would it work?

**Reply by Dhranios** [reply]
It works the same as with singleplayer; ghosts target the closest player; closest player when a ghost dies reels it in.

**Comment by Fireworkstars46** [comment]
So the release 2 version is not for 1.18?

**Reply by Dhranios** [reply]
Correct, command addition from 1.19 is being used, along with NBT changes 1.19 made.
